cmd/jsonnet.cpp

```diff
@@ -618,27 +628,29 @@ int main(int argc, const char **argv)
             return EXIT_FAILURE;
         }
 
-        // Read input files.
-        std::string input;
-        if (!read_input(&config, &input)) {
-            jsonnet_destroy(vm);
-            return EXIT_FAILURE;
-        }
-
         // Evaluate input Jsonnet and handle any errors from Jsonnet VM.
         int error;
         char *output;
         switch (config.cmd) {
             case EVAL: {
+                assert(config.inputFiles.size() == 1);
```

Comment:
There is a check for that just above the one for "ERROR: Already specified".  I didn't need to touch it.